imaging/ml/ml_codelab/breast_density_cloud_ml.ipynb

```diff
@@ -266,7 +135,7 @@
     "id": "aIfKnxqHjE9A"
    },
    "source": [
-    "Next we will convert the DICOMs to JPEGs using the ExportDicomData API. This is an asynchronous call that returns an Operation name. The operation name will be used to poll the status of the operation."
+    "Next we will convert the DICOMs to JPEGs using the ExportDicomData."
```

Comment:
Can you add the link to this: https://cloud.google.com/sdk/gcloud/reference/beta/healthcare/dicom-stores/export/gcs

Reply:
done